### ISSUE TITLE
Deployment fails with approot not found

### ISSUE THREAD

**Issue by abdu292** [Microsoft/BikeSharing360_BackendServices#6]
Any help on this error?

01:48:55 - Warning: Skipping source contentPathLib (C:\dev\BS360_Backend\src\BikeSharing.S
01:48:55 - ervices.Deploy\bin\Debug\staging\BikeSharing.Services.Deploy\pubs\rides\approot
01:48:55 - ) because of rule SkipInvalidSource.
01:48:55 - Could not find file 'C:\dev\BS360_Backend\src\BikeSharing.Services.Deploy\bin\D
01:48:55 - ebug\staging\BikeSharing.Services.Deploy\pubs\rides\approot'.  Learn more at: h
01:48:55 - ttp://go.microsoft.com/fwlink/?LinkId=221672#FileOrFolderNotFound.
01:48:55 - Info: Adding directory (bikesharing-services-rides3slpdvq65s476\node_modules\te
01:48:55 - rraformer\.grunt\grunt-gh-pages\gh-pages\src\partials\index_partials\terraforme
01:48:55 - r_core).
01:48:55 - 
01:48:56 - 
01:48:56 - 
01:48:56 - 
01:48:56 - Deploying template using PowerShell script failed.

**Comment by eiximenis** [comment]
Hi @abdu292 
Do you have installed the node tools for Visual Studio (https://www.visualstudio.com/es/vs/node-js/). They're required to build the Rides API.

**Comment by abdu292** [comment]
@eiximenis Yeah. I do have node tools installed. Though it throws this error I can see the services and all the dbs created on Azure. I am proceeding with the setup. I will post another comment with an update soon.

**Comment by SamVanCutsem** [comment]
@abdu292 Any update on this?

**Comment by abdu292** [comment]
@SamVanCutsem I still have this error showing up. However the deployment was successful on to Azure, and the BikeSharing app on my android can consume the services. As this was working I haven't done anything further to resolve approot not found issue.

**Comment by eiximenis** [comment]
Hi.
I just updated the projects using the latest VS2017 (RC3). Updated projects can be found in the VS2017-RC3 branch.
Not sure if you were using VS2017. If you did, maybe this can solve your issue.

**Comment by abdu292** [comment]
@eiximenis Thank you. Yeah I am using VS2017.
